.github/labeler.yml

```diff
@@ -6,20 +6,15 @@ Python:
   - python/**
   - notebooks/**
   
-libcusignal:
+cpp:
```

Comment:
This repo has no `cpp` label.

Reply:
Do you have an idea what I can update it to?